pkg/controller/volume/selinuxwarning/selinux_warning_controller.go

```diff
@@ -117,7 +120,7 @@ func NewController(
 				Name: "selinux_warning",
 			},
 		),
-		labelCache: volumecache.NewVolumeLabelCache(),
+		labelCache: volumecache.NewVolumeLabelCache(seLinuxTranslator),
 	}
 
 	err := c.vpm.InitPlugins(plugins, prober, c)
```

Comment:
Not visible in this PR, but in following code: 

```
	conflictCh := make(chan cache.Conflict)
	go func() {
		c.cache.SendConflicts(c.logger, conflictCh)
		close(conflictCh)
	}()
```

Is there any value in using channel for reading conflicts from cache? It appears to me, those can be read synchronously...

Reply:
The cache does not have list of conflict ready in the format that metrics need it. I found it  better to structure the cache to react to new / deleted Pods quickly than cache all conflicts.
I'd optimize later, if it's necessary.

Reply:
And the main reason was to avoiding allocation of array of conflicts of unknown size. For large nr. of conflicts, it would need to be extended several times just to discard it after the call.